soccer/Referee.cpp

```diff
@@ -31,7 +31,6 @@ Referee::Referee(Context* const context)
       command_(HALT),
       sent_time{},
       received_time{},
-      stage_time_left{},
```

Comment:
It defaults to 0, but other RJ:Times are in the constructor. We should probably be consistent